opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -109,7 +165,7 @@ def max(self):
 
     @property
     def sum(self):
-        if isinstance(self._instrument, _Monotonic):
+        if self._is_monotonic:
             return self._sum
 
         _logger.warning(
```

Comment:
Sorry i know this isn't from this PR, but can you remove this warning? It's not actually unexpected behavior, its well defined in the spec

Reply:
All right, removing it :v: